### PR TITLE
add crd(podgroup, elasticquotas) field

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
- add crd(podgroup, elasticquotas) field
```bash
➜  bases git:(add_crd_field) ✗ kubectl get elasticquotas.scheduling.x-k8s.io  
NAME   USED                                            MAX                                             AGE
test   {"cpu":"0","memory":"0","nvidia.com/gpu":"0"}   {"cpu":20,"memory":"40Gi","nvidia.com/gpu":2}   3d6h
➜  bases git:(add_crd_field) ✗ kubectl get podgroups.scheduling.x-k8s.io
NAME   PHASE     MINMEMBER   RUNNING   SUCCEEDED   FAILED   AGE
pg1    Running   2           3                              41h

```
#### Which issue(s) this PR fixes:

Fixes https://github.com/kubernetes-sigs/scheduler-plugins/issues/730
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@googs1025 you should generate CRDs via code generation scripts you can control `additionalPrinterColumns` via comments in the `apis/scheduling/v1alpha1/types.go` file. Check:
https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns

**Comment by googs1025** [comment]
> @googs1025 you should generate CRDs via code generation scripts you can control `additionalPrinterColumns` via comments in the `apis/scheduling/v1alpha1/types.go` file. Check: https://book.kubebuilder.io/reference/generating-crd.html#additional-printer-columns

Oh, my bad. I will go ahead and add it. Thank you for your response!

File: manifests/capacityscheduling/crd.yaml [hunk 1]
```diff
@@ -19,7 +19,21 @@ spec:
...
+      type: string
+    - description: Max is the set of desired max limits for each named resource.
+      jsonPath: .spec.max
+      name: Max
+      type: integer
+    - description: Age is the time ElasticQuota was created.
+      jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1alpha1
```
Comment (Huang-Wei) [inline manifests/capacityscheduling/crd.yaml:36]: These fields (3) are not consistent with the fields (6) in config/crd/bases/*

TODO: having 2 copies of the same CRD should be fixed - we should use config/crd/bases/* as the single source of truth, and have all other CRD yaml `ln -s` to config/crd/bases/*

Comment (googs1025) [reply manifests/capacityscheduling/crd.yaml:36]: There are two crd objects, one pggroup adds 6 fields, and one elasticquota adds 3 fields.

Comment (googs1025) [reply manifests/capacityscheduling/crd.yaml:36]: Does this todo item need to be fixed in this pr?

**Comment by googs1025** [comment]
@PiotrProkop Could you please help me review this pr? thanks a lot!

File: apis/scheduling/v1alpha1/types.go [hunk 2]
```diff
@@ -116,6 +119,12 @@ const (
 // +kubebuilder:resource:shortName={pg,pgs}
 // +kubebuilder:subresource:status
 // +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/scheduler-plugins/pull/50"
+// +kubebuilder:printcolumn:name="Phase",JSONPath=".status.phase",type=string,description="Current phase of PodGroup."
+// +kubebuilder:printcolumn:name="MinMember",JSONPath=".spec.minMember",type=integer,description="MinMember defines the minimal number of members/tasks to run the pod group."
+// +kubebuilder:printcolumn:name="Running",JSONPath=".status.running",type=string,description="The number of actively running pods."
+// +kubebuilder:printcolumn:name="Succeeded",JSONPath=".status.succeeded",type=string,description="The number of pods which reached phase Succeeded."
+// +kubebuilder:printcolumn:name="Failed",JSONPath=".status.failed",type=string,description="The number of pods which reached phase Failed."
```
Comment (PiotrProkop) [inline apis/scheduling/v1alpha1/types.go:126]: those shouldnt be of type `integer`?

File: apis/scheduling/v1alpha1/types.go [hunk 3]
```diff
@@ -29,6 +29,9 @@ import (
 // +kubebuilder:resource:shortName={eq,eqs}
 // +kubebuilder:subresource:status
 // +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/scheduler-plugins/pull/52"
+// +kubebuilder:printcolumn:name="Used",JSONPath=".status.used",type=string,description="Used is the current observed total usage of the resource in the namespace."
```
Comment (PiotrProkop) [inline apis/scheduling/v1alpha1/types.go:32]: s/string/integer?

File: apis/scheduling/v1alpha1/types.go (hunk 2 above)
Comment (googs1025) [reply apis/scheduling/v1alpha1/types.go]: my bad.  I looked at the code and it needs to be integer. done

File: apis/scheduling/v1alpha1/types.go (hunk 3 above)
Comment (googs1025) [reply apis/scheduling/v1alpha1/types.go:32]:
```go
// +kubebuilder:printcolumn:name="Used",JSONPath=".status.used",type=string,description="Used is the current observed total usage of the resource in the namespace."
// +kubebuilder:printcolumn:name="Max",JSONPath=".spec.max",type=string,description="Max is the set of desired max limits for each named resource."
```

I set the used max fields to string

File: manifests/capacityscheduling/crd.yaml (hunk 1 above)
Comment (googs1025) [reply manifests/capacityscheduling/crd.yaml]: @Huang-Wei /PTAL

Comment (Huang-Wei) [reply manifests/capacityscheduling/crd.yaml]: > Does this todo need to be fixed in this pr?

We can leave it as a follow-up.

**Review by Huang-Wei — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**